### PR TITLE
Move to npm and remove pug as a dependency

### PR DESCRIPTION
**Pull request by juliushaertl**
- Move to npm instead of yarn to avoid confusion and two lockfiles
- Manually remove pug from the package-lock dependencies since it only seems to be pulled in as one of the dev/optional dependencies of `@vue/component-compiler` https://github.com/vuejs/vue-component-compiler/blob/64213d92fbc4ce2ef724fd3670d9c2f67a371b83/package.json#L33-L60 and is not used by us in any way

### CODE REVIEW

**Comment by juliushaertl** [comment]
node-forge and axios are already up to date in the package-lock.json though not in the yarn.lock for some reason.

**Review by skjnldsv — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ npm install --save @nextcloud/vue-dashboard
 
 ```sh
 npm link
-yarn
-yarn build
+npm ci
+npm rn build
```
Comment (eneiluj) [inline README.md:16]:
```suggestion
npm run build
```

**Review by eneiluj — CHANGES_REQUESTED** [review]
I first thought `rn` was an alias of `run` but it's definitely a typo :grin:.
All good for the rest.

File: README.md (hunk 1 above)
Comment (juliushaertl) [reply README.md]: Woops, good catch :) Fixed

**Review by eneiluj — APPROVED** [review]

**Comment by nickvergessen** [comment]
Time for 1.1 or 2.0 🚀

**Pull request merged**